Makefile

```diff
@@ -68,6 +68,11 @@ license: clean prepareLic
 .PHONY: verify
 verify: clean license test
 
+.PHONE: fmt
+fmt:
+	$(GO_GET) -u github.com/dubbogo/tools/cmd/imports-formatter
+	imports-formatter
+
```

Comment:
Please add this function to a new pull request. Keep a pull request to implement only one function as much as possible to make code review easier.

Reply:
I will create a new pull request for this part.